src/graphql/mutations/password_reset.rs

```diff
@@ -83,6 +85,7 @@ impl PasswordResetMutation {
             ));
         }
         let user_token = schema::password_reset_tokens::Entity::find_by_id(token_id)
+            .filter(password_reset_tokens::Column::ExpiresAt.gt(Utc::now()))
             .one(db)
             .await?
             .ok_or_else(|| new_err("TOKEN_NOT_FOUND", "token doesn't exist"))?;
```

Comment:
This is technically not correct, since it may exist but may just be expired. Should we have a check after this for expiry instead and throw a error "RESET_TOKEN_EXPIRED"?

Reply:
if the token is expired it doesn't exist, it should only check for valid tokens